Src/AutoFixture.xUnit2/MemberAutoDataAttribute.cs

```diff
@@ -67,5 +74,25 @@ protected MemberAutoDataAttribute(DataAttribute autoDataAttribute, string member
         /// Gets the parameters passed to the member. Only supported for static methods.
         /// </summary>
         public IEnumerable<object> Parameters { get; }
+
+        /// <summary>
+        /// Returns the composition of test data from the member on the test class
+        /// combined with the auto-generated data specimens generated by AutoFixture.
+        /// </summary>
+        /// <param name="testMethod">The method that is being tested.</param>
+        /// <returns>
+        /// Returns the composition of the theory data provided by MemberDataAttribute and AutoFixture.
+        /// </returns>
+        public override IEnumerable<object[]> GetData(MethodInfo testMethod)
+        {
+            if (testMethod == null) throw new ArgumentNullException(nameof(testMethod));
+
+            var memberData = this.MemberDataAttribute.GetData(testMethod).ToList();
+
+            return from memberValues in memberData
+                   from autoValues in this.DataAttributeFactory.Invoke().GetData(testMethod).Take(1)
+                   let combinedValues = memberValues.Concat(autoValues.Skip(memberValues.Length))
```

Comment:
I wonder if we could skip the values generation for these method arguments the memberData has already values for. With such improvement we should gain:
- performance boost (we do not generate values we already know we will skip)
- potential generation failure (it may happen that with the MemberAutoData we deliver such data that AutoDataAttribute would fail to generate)

I've already provided such behavior but with custom implementation of [AutoDataAttribute](https://github.com/ObjectivityLtd/AutoFixture.XUnit2.AutoMock/blob/master/src/Objectivity.AutoFixture.XUnit2.Core/Attributes/AutoDataAdapterAttribute.cs) in [AutoFixture.XUnit2.AutoMock](https://github.com/ObjectivityLtd/AutoFixture.XUnit2.AutoMock) packages.